hibikase/LineTimingDecorations.cpp

```diff
@@ -65,6 +61,33 @@ SyllableDecorations::SyllableDecorations(const QPlainTextEdit* text_edit, int st
     setVisible(true);
 }
 
+QColor SyllableDecorations::GetPlayingColor() const
+{
+    return QPalette().color(QPalette::Text);
+}
+
+QColor SyllableDecorations::GetNotPlayedColor() const
+{
+    QColor col(0x55, 0x77, 0x55);
+    // If the system text colour is darker than our colours, make them lighter
+    // (Assumes a system with conventional grayscale text fg/bg colours…)
+    int diff = QPalette().color(QPalette::Text).lightness() - col.lightness();
```

Comment:
Would it be possible to use `>` directly, without any subtraction? Would probably make it a bit easier to read.

Reply:
Yes, but then I'd effectively be comparing the colours twice!!

Reply:
Oh, right, because `diff` is used in the call to `lighter`. I see now.